src/components/pip/pip.scss

```diff
@@ -7,26 +7,32 @@ $hideContainerGap: 5px;
 .childPlayer {
   position: relative;
 
-  .videoContainer {
-    border-radius: 8px;
-    overflow: hidden;
-    box-shadow: 0 0 40px 0 rgba(0, 0, 0, 0.2), 0 0 4px 0 rgba(0, 0, 0, 0.3);
+  &.hovering,
+  &.dragging {
+    .innerButtons {
+      display: none;
+    }
+  }
+  &.hovering .hideContainer {
+    display: none;
+  }
+  &.dragging .hideContainer {
+    visibility: hidden;
   }
 
   .hideContainer {
+    position: relative;
+    width: 66px;
+    height: $hide-container-height;
+    margin-bottom: $hideContainerGap;
```

Comment:
Please change casing of all vars in all scss files to "$xxxx-yyy-zzzzz".

Reply:
done